### DISCUSSION TITLE
小程序端使用pag，wasm 文件过大

### DISCUSSION THREAD

**Discussion by aria-yx** [Tencent/libpag#1829] [Q&A] [answered]
小程序端使用pag需要多加载多一个 wasm 文件，648kb，小程序有包大小限制，相对来说会比较占内存，这个文件可以用什么方式压缩吗？
<img width="763" alt="企业微信截图_32ab8df5-735c-4e48-8676-da41d71e328c" src="https://github.com/Tencent/libpag/assets/33219127/6341f641-eb3e-4551-9d77-d6cea37c27c8">

**Comment by freelancebo** [comment]
+1 无法上传代码了

**Comment by wzbj** [comment]
小程序分包呢，能解决包的限制

**Comment by zhalice2011** [comment, marked as answer]
你好, 当前版本wasm文件已经是最小了.

如果你有好的优化方法, 欢迎提交 PR 共建~
